benchmark/nixlbench/README.md

```
@@ -970,6 +970,42 @@ nvidia-smi topo -m
+CUFILE_ALLOW_COMPAT_MODE=true CUFILE_FORCE_COMPAT_MODE=true \
+nixlbench --backend GDS --initiator_seg_type VRAM --filepath /mnt/storage/testdir \
+  --storage_enable_direct --start_batch_size 1 --max_batch_size 128
+```
+
+In compat mode every in-flight batch entry takes a CPU bounce buffer from the cuFile
+POSIX pool (`posix_pool_slab_count`, default 64 buffers for the 1MiB slab class). When
+the number of concurrent batch entries exceeds the pool size, `cuFileBatchIOSubmit`
+blocks forever waiting for a free buffer. To confirm, set `"logging": {"level": "DEBUG"}`
+in cufile.json and look for:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)